imap/mailbox.c

```diff
@@ -7628,6 +7629,12 @@ EXPORTED void mailbox_iter_startuid(struct mailbox_iter *iter, uint32_t uid)
     iter->recno = uid ? mailbox_finduid(mailbox, uid-1) : 0;
 }
 
+EXPORTED void mailbox_iter_uidset(struct mailbox_iter *iter, struct seqset *seq)
```

Comment:
It's be nicer to user `const seqset *` to make ownership of seq clear. But this will require to update the signature of `seqset_ismember`. The the implementation of ismember could do the usual backdoor cast to non-const.

Reply:
Yeah, that's a great idea.  I'll do that, because I love this API trick.

Reply:
Done and also now 'seqset_t' everywhere.  Also it's `lib/seqset.[ch]` and such.